Lib/test/test_tuple.py

```diff
@@ -42,6 +42,35 @@ def test_keyword_args(self):
         with self.assertRaisesRegex(TypeError, 'keyword argument'):
             tuple(sequence=())
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_keywords_in_subclass(self):
```

Comment:
if you are interested in, `Constructor for PyTuple` probably is a point to start fix this issue